apis/server/container_bridge.go

```diff
@@ -43,6 +43,11 @@ func (s *Server) createContainer(ctx context.Context, rw http.ResponseWriter, re
 	logCreateOptions("container", config)
 
 	name := req.FormValue("name")
+	//consider set specific id by url params
+	specificID := req.FormValue("recover-id")
```

Comment:
Where to define `recover-id`?
Actually I do not find anywhere to define this. @wangforthinker

Reply:
Updated it to param specificID.